### ISSUE TITLE
Ardupilot.org

### ISSUE THREAD

**Issue by OlivierJB** [ArduPilot/ardupilot_wiki#667]
Write FAQ for visitors completely new to Ardupilot and the world of "drones". Simple and non technical.

What Ardupilot is (and is not), how to get started, etc ... Also sort of and introduction to the wiki that it will point to.

**Comment by Hwurzburg** [comment]
@tridge @rmackay9 @OlivierJB  so do we still need this? shouldnt it be part of the Ardupilot.org page...if so its concrete now right? should it be part of the Ardupilot site (Docs target page) as a topic right under HOME in the left side bar?

**Comment by OlivierJB** [comment]
Yes (part of ardupilot.org). Still would be one of those  "nice to have" but many edits elsewhere adding details have made this less of an issue.
